src/patternfly/layouts/Level/styles.scss

```diff
@@ -1,21 +1,16 @@
+@import "../../patternfly-utilities";
+
+$pf-l-level--gutter-size: $pf-global--gutter;
```

Comment:
`gutter-size` is not a CSS property:

`pf-l-level--has-gutter--MarginRight`

Reply:
Done